sklearn/manifold/t_sne.py

```diff
@@ -628,8 +631,8 @@ class TSNE(BaseEstimator):
     def __init__(self, n_components=2, perplexity=30.0,
                  early_exaggeration=12.0, learning_rate=200.0, n_iter=1000,
                  n_iter_without_progress=300, min_grad_norm=1e-7,
-                 metric="euclidean", init="random", verbose=0,
-                 random_state=None, method='barnes_hut', angle=0.5):
+                 metric="euclidean", metric_params = {}, init="random",
```

Comment:
Please use `None` as default.

Reply:
and remove whitespaces around the `=` sign, to comply with PEP8